layer/vulkan_reflex_layer.cpp

```diff
@@ -878,24 +808,27 @@ struct VkDeviceOverrides {
         TRACE("(%p, %p { %s })",
             queue, pQueueTypeInfo, vkroots::helpers::enumString(pQueueTypeInfo->queueType));
 
-        pDispatch->QueueNotifyOutOfBandNV(queue, pQueueTypeInfo);
+        dispatch.QueueNotifyOutOfBandNV(queue, pQueueTypeInfo);
 
         if (!injectFrameIDs)
             return;
 
-        if (auto [queueContext, deviceContext] = ::GetContexts(queue); queueContext) {
-            switch (pQueueTypeInfo->queueType) {
-                case VK_OUT_OF_BAND_QUEUE_TYPE_RENDER_NV:
-                    queueContext->outOfBandRenderSubmit = true;
-                    break;
-                case VK_OUT_OF_BAND_QUEUE_TYPE_PRESENT_NV:
-                    queueContext->outOfBandPresent = true;
-                    break;
-                default:
-                    break;
-            }
+        if (!dispatch.UserData)
+            dispatch.UserData.emplace<ReflexQueueContextData>();
```

Comment:
Is this guaranteed to be zero-initialized? I don't remember how `std::any` works exactly but I _think_ there's a nonzero chance that our `Reflex{Device,Queue}ContextData` types stored in `UserData` may need something like parameterless constructors for this to be safe.